extensions/2.0/Vendor/MPEG_accessor_timed/README.md

```diff
@@ -0,0 +1,106 @@
+
+glTF extension to specify timed accessor format formats
+
+
+**`MPEG_accessor_timed` Properties**
+
+|   |Type|Description|Required|
+|---|---|---|---|
+|**immutable**|`boolean`|This flag indicates if the accessor information may change over time|No, default: `true`|
+|**bufferView**|`integer`|This property provides the index in the bufferViews array to a bufferView element that points to the timed accessor information header. |No|
+|**suggestedUpdateRate**|`number`|This property provides the frequency at which the renderer is recommended to poll the underlying buffer for new data..|No, default: `25`|
+|**extensions**|`object`|JSON object with extension-specific objects.|No|
+|**extras**|[`any`](#reference-any)|Application-specific data.|No|
+
+Additional properties are allowed.
+
+* **JSON schema**: [MPEG_accessor_timed.schema.json](/Extensions/MPEG_accessor_timed/schema/MPEG_accessor_timed.schema.json)
+
+### MPEG_accessor_timed.immutable
+
+This flag equal to false indicates the accessor information componentType, type, and normalize may change over time. The changing values of componentType, type and normalize are provided through accessor information header. This flag equal to true indicates the accessor information componentType, type, and normalize do not change over time and are not present in the accessor information header. Note that count, max, min, and byteOffset are expected to change and are always included as part of the timed accessor information header.
+
+* **Type**: `boolean`
+* **Required**: No, default: `true`
+
+### MPEG_accessor_timed.bufferView
+
+This property provides the index in the bufferViews array to a bufferView element that points to the timed accessor information header as described in ISO/IEC 23090-14:Table 8. byteLength field of the bufferView element indicates the size of the timed accessor information header. The buffer properties in the bufferView element shall point to the same buffer as the bufferView in the containing accessor object. In the absence of the bufferView attribute, it shall be assumed that the buffer has no dynamic header. In that case, the immutable flag shall be present and shall be set to True.
+
+* **Type**: `integer`
+* **Required**: No
+* **Minimum**: ` >= 0`
+
+### MPEG_accessor_timed.suggestedUpdateRate
+
+The suggestedUpdateRate provides the frequency at which the renderer is recommended to poll the underlying buffer for new data. The rate is provided in number of changes per second.
+
+* **Type**: `number`
+* **Required**: No, default: `25`
+* **Minimum**: ` >= 0`
+
+### MPEG_accessor_timed.extensions
+
+JSON object with extension-specific objects.
+
+* **Type**: `object`
+* **Required**: No
+* **Type of each property**: Extension
+
+### MPEG_accessor_timed.extras
+
+Application-specific data.
+
+* **Type**: [`any`](#reference-any)
+* **Required**: No
+
+
+## Known Implementations
+
+* [ISO/IEC WD 23090-24](https://www.iso.org/standard/83696.html)
+## Resources
```

Comment:
Missing a line break before the header (also in other markdown files).

Reply:
implemented.